libs/gi/dm/src/dm/character/AvatarSkillDepotExcelConfigData.ts

```diff
@@ -41,16 +41,16 @@ export type AvatarSkillDepotExcelConfigData = {
  * const#     AvatarSkillExcelConfigData
  */
 
-const avatarSkillDepotExcelConfigDataSrc = JSON.parse(
-  readDMJSON('ExcelBinOutput/AvatarSkillDepotExcelConfigData.json')
+const avatarSkillDepotExcelConfigDataSrc = readExcelJSON(
+  'ExcelBinOutput/AvatarSkillDepotExcelConfigData.json'
 ) as AvatarSkillDepotExcelConfigData[]
 
 const avatarSkillDepotExcelConfigData = Object.fromEntries(
   avatarSkillDepotExcelConfigDataSrc
     .map((skill) => {
       //FIXME: custom processing because there are empty objects in here. for RaidenShogun only...
       if (skill.id === 5201) {
-        const [k1, k2, , , k5] = skill.inherentProudSkillOpens
+        const [k1, k2, , k5] = skill.inherentProudSkillOpens
```

Comment:
This will probably break when we move to a version where dim doesn't delete zeros. We could look for better ways to detect the type of the passive skill to avoid this